app/views/componentsP5/table_1/row_3.scala.xml

```diff
@@ -1,52 +1,53 @@
 @import views.xml.componentsP5.table_1.consignee
-@import views.xml.componentsP5.table_1.total_gross_mass
-
-@(ie029Data: models.P5.departure.IE029Data)
-
-<fo:table-row height="8mm">
-
-  <fo:table-cell>
-    <fo:block/>
-  </fo:table-cell>
-
-  <fo:table-cell text-align="center" vertical-align="top" border-left-style="solid" border-left-width="1pt" border-left-color="black">
-    <fo:block/>
-  </fo:table-cell>
-
-  @consignee(ie029Data.consignee, ie029Data.consigneeIdentificationNumber)
-
-  <fo:table-cell>
-    <fo:block>
-      <fo:table width="100%" table-layout="fixed">
-        <fo:table-column/>
-
-        <fo:table-body>
-
-          <fo:table-row>
-            <fo:table-cell>
-              <fo:block>
-                <fo:table width="100%" table-layout="fixed">
-                  <fo:table-column column-width="16mm"/>
-                  <fo:table-column column-width="22mm"/>
-                  <fo:table-column column-width="28.5mm"/>
-                  <fo:table-column column-width="28.5mm"/>
-
-                  <fo:table-body>
-                    <fo:table-row height="8mm">
-                      @total_items(ie029Data.totalItems)
-                      @total_packages(ie029Data.totalPackages)
-                      @total_gross_mass(ie029Data.totalGrossMass)
-                      @security(ie029Data.security)
+        @import views.xml.componentsP5.table_1.total_gross_mass
+
+        @(viewModel: viewmodels.P5.TableViewModel)
+
+<fo:table-row height="8%">
+
+    <fo:table-cell>
+        <fo:block/>
+    </fo:table-cell>
+
+    <fo:table-cell text-align="center" vertical-align="top" border-left-style="solid" border-left-width="1pt"
+                   border-left-color="black">
+        <fo:block/>
+    </fo:table-cell>
+
+    @consignee(viewModel.table1ViewModel.consignee, viewModel.table1ViewModel.consigneeIdentificationNumber)
+
+    <fo:table-cell>
+        <fo:block>
+            <fo:table width="100%" table-layout="fixed">
+                <fo:table-column/>
+
+                <fo:table-body>
+
+                    <fo:table-row>
+                        <fo:table-cell>
+                            <fo:block>
+                                <fo:table width="100%" table-layout="fixed">
+                                    <fo:table-column column-width="31.3%"/>
+                                    <fo:table-column column-width="43.08%"/>
+                                    <fo:table-column column-width="43.08%"/>
+                                    <fo:table-column column-width="43.08%"/>
+
```

Comment:
Shouldnt these add up to 100%, these appear to be random values?

Reply:
I did have to amend the last 3 columns but still doesnt add to 100%
![image](https://github.com/hmrc/transit-movements-trader-manage-documents/assets/31237148/ec589313-5ac4-403c-919d-832b75f27e31)